Bottom: 53e9f6317d40df0fa44bc02a65e9ef96ad621ed4
Top:    7d175de3831294af6e84b47bee0062bafadb97bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 01:37:12 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
